SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -247,6 +247,43 @@ void CTurbSSTSolver::Postprocessing(CGeometry *geometry, CSolver **solver_contai
   }
   END_SU2_OMP_FOR
 
+
+  /*--- Compute turbulence index ---*/
+  if(config->GetKind_Trans_Model() != TURB_TRANS_MODEL::NONE) {
+    auto* flowNodes = su2staticcast_p<CFlowVariable*>(solver_container[FLOW_SOL]->GetNodes());
+
+    for (auto iMarker = 0; iMarker < config->GetnMarker_All(); iMarker++)
+      if (config->GetViscous_Wall(iMarker)) {
+          SU2_OMP_FOR_STAT(OMP_MIN_SIZE)
+          for (auto iVertex = 0u; iVertex < geometry->nVertex[iMarker]; iVertex++) {
+            const auto iPoint = geometry->vertex[iMarker][iVertex]->GetNode();
```

Comment:
Too much indentation.

Reply:
How should I solve this?

Reply:
The line `SU2_OMP_FOR_STAT(OMP_MIN_SIZE)` is indented 4 spaces, and our standard is 2 spaces